compose.yaml

```diff
@@ -94,7 +94,7 @@ services:
   openc3-cosmos-cmd-tlm-api:
     # For rootless podman - Uncomment this user line and comment out the next
     # user: 0:0
-    user: "${OPENC3_USER_ID}:${OPENC3_GROUP_ID}"
+    user: "${OPENC3_USER_ID:-1000}:${OPENC3_GROUP_ID:-1000}"
```

Comment:
Why the change to hard code the user and group ids?

Reply:
Its placing the default values of 1000 here instead of in the .env.  If openc3.sh sets the values that wins over the default.